examples/scenarios/test_crop/data/sequences/single_day.csv

```diff
@@ -0,0 +1,25 @@
+timeindex,vwc_rzd,ghi,t_air,e
```

Comment:
vwc_rzd means vwc has already been multiplied by rzd I guess? This need to be clarified to ask the right inputs

Reply:
era5 provides various vwc values in various depths (as far as i remember, 0 -2m); vwc_rzd is the averaged vwc in this upper soil layer, roughly corresponding to root zone depth (rzd)